src/views/Main.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import { fetchCountries } from '../services/countries';
+import Country_Card from '../components/Country_Card/Country_Card';
+import Dropdown from '../components/Search/Dropdown';
+import Input from '../components/Search/Input';
+import './Main.css';
+
+export default function Main() {
+  const [countries, setCountries] = useState([]);
+  const [continents, setContinents] = useState([]);
+  const [continent, setContinent] = useState('All');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [searchBar, setSearchBar] = useState('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const resp = await fetchCountries();
+      {
+        try {
+          // I realized after doing all the research for this that it was totally unnecessary since we only have 7 things to choose from
+          //make an array with the continents only once
+          const onlyContinents = [...new Set(resp.map((aContinent) => aContinent.continent))];
+          //filter out the null values and add All to the start
+          const filteredContinents = onlyContinents.filter(Boolean);
```

Comment:
nice work here thinking through the logic, even though it felt 'totally unnecessary' -- the work behind it in the important part